### ISSUE TITLE
a small error with the Spanish translation of nvda

### ISSUE THREAD

**Issue by ionutsoftware** [nvaccess/nvda#17567]
hello,
good morning, afternoon or evening.
apparently, there is a small error with the translation of NVDA to English, which occurs when pressing the NVDA key + the DEL key, which is used to report the position of the current object or text on the screen. the error is as follows: "Object edges positioned 4.6 percent from left edge of screen, 16.0 percent from top edge of screen, width is 74.2 percent of screen, height is 55.5 porciento de la pantalla" .
as you can see, the error comes in that the sentence is in english, except ‘de la pantalla’, which is in spanish as it should be. it occurs when you press the command 2 times.
- specifications of the installation environment:
  - **Version of NVDA**: 2024.1.1
  - Windows**: Windows11
  Version 23H2 (OS build 22631.4602)
Greetings.

**Comment by josephsl** [comment]
CC @nvdaes and other Spanish translation folks

**Comment by nvdaes** [comment]
Hello. Let's close this issue since, per the [document about reporting issues](https://github.com/nvaccess/nvda/blob/master/projectDocs/issues/readme.md), this should be reported to the translations mailing list:

> Issues with translations should be reported to the [NVDA Translators list.](https://groups.io/g/nvda-translations).

**Comment by nvdaes** [comment]
I'll post a message to the translators mailing list.

**Comment by nvdaes** [comment]
For reference, see this topic on the translators mailing list:

https://groups.io/g/nvda-translations/message/4466
